api/auth/jwtUtils.test.js

```diff
@@ -69,13 +69,13 @@ tap.test('jwtUtils', async t => {
     const scenarios = [
       [
         '/apds/1/files/12345',
-        'okta-token-storage_accessToken={%22accessToken%22%3A%22example.cookie.value%22%2C%22max-age%22%3A900}',
+        'gov.cms.eapd.api-token={%22accessToken%22%3A%22example.cookie.value%22%2C%22max-age%22%3A900}',
         'example.cookie.value',
         'returns the JWT'
       ],
       [
         '/apds/1/files/12345',
-        'okta-token-storage_ACCESSTOKEN={%22accessToken%22%3A%22example.cookie.value%22%2C%22max-age%22%3A900}',
+        'gov.cms.eapd.API-TOKEN={%22accessToken%22%3A%22example.cookie.value%22%2C%22max-age%22%3A900}',
         'example.cookie.value',
         'returns the JWT'
```

Comment:
Can you add a comment explaining the JWT?

Reply:
I'm sorry, I just saw this. A JWT is a java web token, it's used for authentication.